antlr4-maven-plugin/pom.xml

```diff
@@ -1,348 +1,151 @@
-                <version>2.4.2</version><!--$NO-MVN-MAN-VER$-->
-                <configuration>
-                    <arguments>-Psonatype-oss-release ${release.arguments}</arguments>
-                </configuration>
-            </plugin>
-		</plugins>
-
-	</build>
-
-	<reporting>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-plugin-plugin</artifactId>
-				<version>3.3</version>
-			</plugin>
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>2.9</version>
-				<configuration>
-					<quiet>true</quiet>
-				</configuration>
-			</plugin>
-
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-jxr-plugin</artifactId>
-				<version>2.3</version>
-			</plugin>
-		</plugins>
-	</reporting>
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.antlr</groupId>
+    <artifactId>antlr4-master</artifactId>
+    <version>4.5.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>antlr4-maven-plugin</artifactId>
+  <packaging>maven-plugin</packaging>
+  <name>ANTLR 4 Maven plugin</name>
+  <description>Maven plugin for ANTLR 4 grammars</description>
+
+  <prerequisites>
+    <maven>3.0</maven>
+  </prerequisites>
+
+  <!-- Ancilliary information for completeness -->
+  <inceptionYear>2009</inceptionYear>
+
+  <!-- ============================================================================= -->
+
+  <!-- What are we depedent on for the Mojos to execute? We need the plugin 
+    API itself and of course we need the ANTLR Tool and runtime and any of their 
+    dependencies, which we inherit. The Tool itself provides us with all the 
+    dependencies, so we need only name it here. -->
+  <dependencies>
+
+    <!-- The things we need to build the target language recognizer -->
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-plugin-api</artifactId>
+      <version>3.0.5</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-project</artifactId>
+      <version>2.2.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-compiler-api</artifactId>
+      <version>2.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.sonatype.plexus</groupId>
+      <artifactId>plexus-build-api</artifactId>
+      <version>0.0.7</version>
+    </dependency>
+    <!-- The version of ANTLR tool that this version of the plugin controls. 
+      We have decided that this should be in lockstep with ANTLR itself, other 
+      than -1 -2 -3 etc patch releases. -->
+    <dependency>
+      <groupId>org.antlr</groupId>
+      <artifactId>antlr4</artifactId>
+      <version>4.6-SNAPSHOT</version>
```

Comment:
:exclamation: `4.6-SNAPSHOT` doesn't seem to exist?

:question: Why not `${project.version}`